CODEOWNERS

```diff
@@ -79,3 +79,6 @@
 /tests/runtime/out_kinesis.c    @pettitwesley
 /tests/runtime/out_opensearch.c @pettitwesley @edsiper
 /tests/runtime/out_s3.c         @pettitwesley
+
+# Devcontainer
+/.devcontainer                  @bkayranci
```

Comment:
We need more than just you I think, let's add me and @niedbalski too plus @edsiper

Reply:
sorry I could not understand, should I remove my username?